src/osha/oira/client/browser/client.py

```diff
@@ -299,7 +299,7 @@ def __call__(self):
 
         self.success = False
 
-        email_regex = re.compile(r"[A-Za-z0-9@.]*")
+        email_regex = re.compile(r"[A-Za-z0-9@._-]*")
```

Comment:
Is there even a point in having this check? 
It is anyway not a good one, emails can also have unicode characters and if the regexp fails there is no proper message to the user.
It sounds like an unrequested feature.

Reply:
It was meant as a bit of extra security because the email address is fed into a database query. But it may be safe to remove because sqlalchemy already does some great things to prevent sql injection...

Reply:
I could change it to a check that there is no whitespace in the address. That would also prevent any sql to work I think. Or I can remove the checks if you think that's better.

Reply:
I would not be worried by an SQL injection attack here. As you mentioned passing it to the sqlalchemy query should be safe